package.json

```
@@ -8,9 +8,20 @@
 		"discord.js": "^11.5.1",
 		"node-cron": "^2.0.3"
 	},
-	"devDependencies": {},
+	"devDependencies": {
+		"eslint": "^6.8.0",
+		"eslint-config-standard": "^14.1.0",
+		"eslint-plugin-import": "^2.20.0",
+		"eslint-plugin-node": "^11.0.0",
+		"eslint-plugin-promise": "^4.2.1",
+		"eslint-plugin-standard": "^4.0.1",
+		"prettier": "^1.19.1"
+	},
 	"scripts": {
 		"start": "node src/index.js",
+		"lint": "eslint \"src/**/*.js\"",
+		"lint:fix": "eslint \"src/**/*.js\"",
+		"format": "prettier --write \"src/**/*.js\"",
```

A couple of things I would suggest here:
1. `format` => `fmt`, because shorter is better.
2. `fmt:check` so I can finally write some CI stuff for this project.
3. let prettier reformat more type of files.

```suggestion
		"fmt": "prettier --write \"./**/*.{js,json,md}\"",
		"fmt:check": "prettier --list-different \"./**/*.{js,json,md}\"",
```

---

I approve this changes, Linting is important, I noticed that they made a lot of changes on the index.js regarding that, made everything more consistent IMO.

---

I totally agree with the consistency! (I would be personally tempted to add the `eslint-config-airbnb-base` plugin to this project :p)